### DISCUSSION TITLE
How to remove from Field Array

### DISCUSSION THREAD

**Discussion by AdamLundberg** [react-hook-form/react-hook-form#4554] [Q&A]
I'm working on a signup form where we have the regular fields like firstName and lastName but I also need to do a dynamic array. The backend is java spring and it is also expecting a list. So I need the data in submit to have an array filled with just the content, no "name":

it should look something like this when click submit 

`"experience": [
                 "McDonalds",
                 "Wall-Mart"
               ]  `

not 

`"experience": [                 
                {                    
                     "name": "McDonalds"
                 },
                 {
                     "name": "Wall-Mart"
                 }
            ]  `

I have managed to follow this [FieldArray](https://github.com/react-hook-form/react-hook-form/blob/master/examples/FieldArray.tsx) and got it working as I wanted. 

But now I need to be able to remove a field, and all information I have found is regarding objects and I'm having some problems understanding it all.

**Comment by bluebill1049** [comment]
isn't that just use the `remove` api? provide a codesandbox if that's not the answer and with steps to reproduce.
